### DISCUSSION TITLE
How do input LONG text??

### DISCUSSION THREAD

**Discussion by BarfingLemurs** [oobabooga/text-generation-webui#410] [Q&A]
I want to use llama instead of chatgpt to summarize an article: https://www.nintendolife.com/reviews/wiiu/yoshis_woolly_world

If you don't have a chatgpt account, use several huggingface spaces for free without signing up like this: https://huggingface.co/spaces/ZeroTech/ChatGPT

<details><summary>My Input:</summary>

```
Below is an instruction that describes a task. Write a response that appropriately completes the request.
### Instruction:
Classify the sentiment of each paragraph and provide a summary of the following text as a json file:

Nintendo has long been the leading light in the platforming genre, a part of that legacy being the focus of Super Mario Anniversary celebrations this year. It's a genre with plenty of scope for variety, however, which is something Nintendo pushed forward with the superb Super Mario World 2: Yoshi's Island on Super NES, placing Mario's sidekick(s) at the core of the adventure with a different approach. Flutter jumping, eating and egg throwing brought a more sedate, puzzle-based approach to running and jumping.

Of course, Kirby was another early innovator in platforming, and was the first of Nintendo's heroes to be turned into yarn by Good-Feel, purveyors of cuteness and clever design. Kirby's Epic Yarn on Wii was a stand-out, and the same core treatment has been applied to Yoshi's Woolly World, with an improved HD engine and a fresh approach. Unlike Kirby's adventure in which the pink one was bolted into what was originally to be a new IP, Yoshi feels like he belongs in this bouncy, soft world; it's a natural fit.

As for storyline, those that would rather not look after Baby Mario should breathe easy, as that mechanic is nowhere to be found here. The plot revolves around a fabric-based world with Kamek appearing and effectively stealing the population of woollen Yoshis, bundling them up into Yarn. Your hero naturally goes in pursuit across a fairly standard stretch of worlds and levels, but it's a cute setup to charm older players and make younger players smile.
Yoshi's Woolly World Review - Screenshot 2 of 6

Upon setting off with Yoshi, Epic Yarn fans will immediately notice that Good-Feel has ramped up the tempo to suit our boot-wearing mascot. A quick build-up of speed sees your feet transform into wheels, and while the general action is a little slower than a 2D Mario title, it's well ahead of Kirby's Epic Yarn. In various senses the core design ideas are similar or the same as its Yarn-based predecessor, but this infusion of pace is welcome. Even Yoshi's jump is more rapid, eschewing the floaty leap of Kirby yet introducing the delightfully frantic flutter jump - for fans of logical (and we use the term loosely) game physics this makes sense, as there's a lightness to Yoshi that suits his woollen construction.

Within the woollen world most of Yoshi's core mechanics remain in place. You can eat enemies and either spit them out or swallow them into woollen balls, the latter of which can then be thrown; as per tradition you can use a quick aim in which the reticule moves in a 180 degree arc, or you can pause for your shot. There is a Wii Remote control option that supports tilting as a means of aiming, but even so the relative complexity of Yoshi's move-set will be a challenge to some familiar with the simple grab and throw of Kirby's Epic Yarn. Such is the nature of early levels and unlimited lives, however, that any player can take the time to master the controls.

For veterans, of course, these controls are a breeze, and the happy news is that Woolly World not only puts Yoshi's abilities to the test, but it also makes for some clever design. Question mark clouds are deviously hidden, and you need to always be on the lookout for loose threads on which to pull or suspicious sections that may be hidden areas. The challenge in Woolly World comes not from simply beating a stage, but completing each requirement of collectibles or proof of skill - you need to end each level with full health, collect all Miiverse stamps (which are hidden within gems), grab five flowers and also five balls of yarn. You may grab them all first time in the earliest levels, but completionists will be drawn into repeating stages on multiple occasions as the difficulty ramps up.
Yoshi's Woolly World Review - Screenshot 3 of 6

It's in stage design, utilising all of these factors, that Good-Feel has achieved some wonderful results. We see riffs on toys and yarn in various forms, and the logic (such as it is) quickly embeds itself into gameplay - hurling balls of wool can fill in outlined platforms, while some enemies can only be defeated by wrapping them in yarn with a projectile and following up with a stomp. You see Yoshi's nose squish into fabric when pushing a hidden wall, or you unravel a blocked path with his tongue, while there are stages that play with invisibility and much more besides. A couple of fast-paced stages even brought franchises such as Donkey Kong Country to mind, some are labyrinths and others are focused on verticality. There's some delightfully playful creativity on show.

This also applies to short challenge areas in which Yoshi transforms into various creatures and items, ranging from an agile dolphin, a steady umbrella, a fast motorbike and more besides. These come in short bursts, and some can be missed entirely by inattentive players; nevertheless they provide some truly memorable moments.

Two player local co-op is included, as is the norm with Nintendo platformers, and this feels like one of the stronger uses of the option. It does have an element of playful chaos as you bounce off each other, which is given a delightful malleability due to the woollen effect, but it's also hugely practical for taking on tougher stages or reaching tricky collectibles. This is due to the fact that you can swallow and turn each other into eggs / wool balls, and therefore use the other player as a projectile weapon, carry them through tricky sections or simply hurl them onto an out-of-reach platform. While we believe all collectibles are there to be claimed in single player, co-op is a simple, fun way to reach areas through team work.
Yoshi's Woolly World Review - Screenshot 4 of 6

That could be useful if a skilful player wants to help out a rookie, then, and the Mellow mode is another option. Switchable from Classic mode at any point - even mid-level - this gives Yoshi the ability to fly at will, replacing the frantic flutter jump with a gentle flight. It also slows down the aiming reticule, too, so it's a useful tool.

With progress in the game you also unlock a wide range of Power Badges, with abilities that vary from drawing items to you magnetically, to calling in cute assist character Poochy, making you invincible to fire and more. Though you accumulate plenty of gems in the game, these badges are nevertheless relatively pricey within the in-game economy, and despite the odd freebie the sense is that they're a backup to help you through tough stages or, perhaps, when trying to hunt down elusive flowers.

Some of the Power Badges absolutely become necessary when hunting collectibles. As we mentioned above, completionists will be kept very busy, and as per the Nintendo tradition collecting also unlocks special levels - these come from grabbing all the flowers in each of the six respective worlds. These extra stages are particularly tough, controller throwing tests, pushing capabilities and stripping away reassuring checkpoints. Unlocking and beating these could add a lot of playtime, while grabbing all five balls of Yarn in each level unlocks a neat Yoshi design.
Yoshi's Woolly World Review - Screenshot 5 of 6

More Yoshi designs, some of the best in fact, come through amiibo. Practically all amiibo are supported, even those that seem inconceivable at first - want a Shulk-based Yoshi? No problem. Bowser, or Toad? All included, even the Splatoon inkling. Collecting and playing with different Yoshi designs is certainly enhanced by amiibo. As for the dedicated Yarn Yoshi, we didn't have one for review but it's been shown how this essentially drops in as a clone Yoshi in single player, which makes some aspects of co-op - such as throwing another player to inaccessible areas - possible while playing alone. These are neat extra features utilising the figures, and a little more inclusive and charming than some throwaway ideas we've seen in other games.

As for playtime, Yoshi's Woolly World is pretty generous for a platformer. In an era when many of us want more bang for our buck, Yoshi's Woolly World delivers - rushing through the campaign should take most at least 8-10 hours, but will likely take much longer with co-op sessions and the aforementioned collect-a-thon. Though the basic level count - excluding extra stages - is 48, these can often be lengthy levels full of puzzles and exploration. There are some quicker levels, but some will genuinely stump you at points, and the use of yarn in puzzles that are based on colours or quirky mechanics keep things fresh. As per the Yoshi tradition - especially the classic SNES entry - you go at a slower, more deliberate pace from start to finish than others such as Mario; mad dashes are the exception, not the rule.
Yoshi's Woolly World Review - Screenshot 6 of 6

Of course, with Kirby's Epic Yarn being quite a looker on Wii, thoughts also turn to the visuals of Yoshi's Woolly World. That's a simple judgement from our perspective - it looks gorgeous. There's lovely detail in seeing threads of yarn, environments deliver welcome variety, and there are clever animations and touches that play into Yoshi's ability to transform; Good-Feel shows terrific attention-to-detail and care. The framerate clicks along at a solid 60fps for the most part, too, though we saw a smidge of slowdown in a few moments. Sound is also excellent with some catchy music, though we did notice some tracks appearing a little more regularly than expected.

Yoshi's Woolly World has very few weak spots, though they should be acknowledged. Boss fights, while a visual spectacle, are slightly hit and miss in design and difficulty, with a couple of strong examples let down by repeats or the occasional sense of niggling disappointment. The slow, methodical nature of some stages also makes some of the checkpoints a little too far apart, with moments where you'll be repeating slightly fiddly puzzles before facing a dreaded tricky section. These are rare and relatively minor complaints that do little to detract from the brilliance of the overall product, but deprive it of pure platforming perfection.
### Response:
```

</details>

When I input the prompt here, it does nothing. It works with shorter things, though.
When I input the prompt in chatGPT demos, it gives output, like this:

<details><summary>ChatGPT Output:</summary>

The sentiment of the first paragraph is positive, with the writer praising Nintendo for their contributions to the platforming genre and highlighting the unique features of Super Mario World 2: Yoshi's Island. The sentiment of the second paragraph is also positive, with the writer highlighting the strengths of Yoshi's Woolly World, its engaging storyline, and engaging game mechanics. They also mention the creative stage design and the challenge presented by collecting various items throughout the game. 

As for the summary, it can be provided as a JSON file containing the following details:
```
{
  "title": "Yoshi's Woolly World Review",
  "author": "Nintendo Life",
  "sentiment":
  {
    "paragraph1": "positive",
    "paragraph2": "positive"
  },
  "summary":
  {
    "genre": "platforming",
    "features":
    [
      "focus on sidekicks",
      "puzzle-based gameplay",
      "plenty of collectibles"
    ],
    "storyline": "Kamek has stolen and bundled up woollen Yoshis into yarn. The player controls a woollen Yoshi through a standard stretch of worlds and levels to rescue the population.",
    "gameplay":
    {
      "core mechanics":
      [
        "eating enemies",
        "spitting them out/swallowing them into woollen balls",
        "quick aim feature",
        "pausable shot",
        "Yoshi's flutter jumps"
      ],
      "special features":
      [
        "fast-paced stages",
        "short challenge areas",
        "two-player local co-op",
        "ability to fly at will",
        "unlockable Yoshi designs with the use of amiibo"
      ]
    },
    "visuals": "Yoshi's Woolly World looks gorgeous, with lovely details in seeing threads of yarn, varied environments, and clever animations."
  }
}
```

</details>

I want to know my token length accurately, is there something online that lets me count how much i'm putting in??

**Comment by BarfingLemurs** [comment]
I had to delete 3 paragraphs, I ran with the 30B instruct GPTQ lora. It turns out that originally I used too much words, and exceeded context length. 
This should give ideas on the maximum amount of words you can summarize with llama currently.
<details><summary> Outputs: </summary>

Below is an instruction that describes a task. Write a response that appropriately completes the request.

### Instruction:
Provide a 3 sentence summary of the following text:

Nintendo has long been the leading light in the platforming genre, a part of that legacy being the focus of Super Mario Anniversary celebrations this year. It's a genre with plenty of scope for variety, however, which is something Nintendo pushed forward with the superb Super Mario World 2: Yoshi's Island on Super NES, placing Mario's sidekick(s) at the core of the adventure with a different approach. Flutter jumping, eating and egg throwing brought a more sedate, puzzle-based approach to running and jumping.

Of course, Kirby was another early innovator in platforming, and was the first of Nintendo's heroes to be turned into yarn by Good-Feel, purveyors of cuteness and clever design. Kirby's Epic Yarn on Wii was a stand-out, and the same core treatment has been applied to Yoshi's Woolly World, with an improved HD engine and a fresh approach. Unlike Kirby's adventure in which the pink one was bolted into what was originally to be a new IP, Yoshi feels like he belongs in this bouncy, soft world; it's a natural fit.

As for storyline, those that would rather not look after Baby Mario should breathe easy, as that mechanic is nowhere to be found here. The plot revolves around a fabric-based world with Kamek appearing and effectively stealing the population of woollen Yoshis, bundling them up into Yarn. Your hero naturally goes in pursuit across a fairly standard stretch of worlds and levels, but it's a cute setup to charm older players and make younger players smile.
Yoshi's Woolly World Review - Screenshot 2 of 6

It's in stage design, utilising all of these factors, that Good-Feel has achieved some wonderful results. We see riffs on toys and yarn in various forms, and the logic (such as it is) quickly embeds itself into gameplay - hurling balls of wool can fill in outlined platforms, while some enemies can only be defeated by wrapping them in yarn with a projectile and following up with a stomp. You see Yoshi's nose squish into fabric when pushing a hidden wall, or you unravel a blocked path with his tongue, while there are stages that play with invisibility and much more besides. A couple of fast-paced stages even brought franchises such as Donkey Kong Country to mind, some are labyrinths and others are focused on verticality. There's some delightfully playful creativity on show.

This also applies to short challenge areas in which Yoshi transforms into various creatures and items, ranging from an agile dolphin, a steady umbrella, a fast motorbike and more besides. These come in short bursts, and some can be missed entirely by inattentive players; nevertheless they provide some truly memorable moments.

Two player local co-op is included, as is the norm with Nintendo platformers, and this feels like one of the stronger uses of the option. It does have an element of playful chaos as you bounce off each other, which is given a delightful malleability due to the woollen effect, but it's also hugely practical for taking on tougher stages or reaching tricky collectibles. This is due to the fact that you can swallow and turn each other into eggs / wool balls, and therefore use the other player as a projectile weapon, carry them through tricky sections or simply hurl them onto an out-of-reach platform. While we believe all collectibles are there to be claimed in single player, co-op is a simple, fun way to reach areas through team work.
Yoshi's Woolly World Review - Screenshot 4 of 6

That could be useful if a skilful player wants to help out a rookie, then, and the Mellow mode is another option. Switchable from Classic mode at any point - even mid-level - this gives Yoshi the ability to fly at will, replacing the frantic flutter jump with a gentle flight. It also slows down the aiming reticule, too, so it's a useful tool.

With progress in the game you also unlock a wide range of Power Badges, with abilities that vary from drawing items to you magnetically, to calling in cute assist character Poochy, making you invincible to fire and more. Though you accumulate plenty of gems in the game, these badges are nevertheless relatively pricey within the in-game economy, and despite the odd freebie the sense is that they're a backup to help you through tough stages or, perhaps, when trying to hunt down elusive flowers.

Some of the Power Badges absolutely become necessary when hunting collectibles. As we mentioned above, completionists will be kept very busy, and as per the Nintendo tradition collecting also unlocks special levels - these come from grabbing all the flowers in each of the six respective worlds. These extra stages are particularly tough, controller throwing tests, pushing capabilities and stripping away reassuring checkpoints. Unlocking and beating these could add a lot of playtime, while grabbing all five balls of Yarn in each level unlocks a neat Yoshi design.
Yoshi's Woolly World Review - Screenshot 5 of 6

More Yoshi designs, some of the best in fact, come through amiibo. Practically all amiibo are supported, even those that seem inconceivable at first - want a Shulk-based Yoshi? No problem. Bowser, or Toad? All included, even the Splatoon inkling. Collecting and playing with different Yoshi designs is certainly enhanced by amiibo. As for the dedicated Yarn Yoshi, we didn't have one for review but it's been shown how this essentially drops in as a clone Yoshi in single player, which makes some aspects of co-op - such as throwing another player to inaccessible areas - possible while playing alone. These are neat extra features utilising the figures, and a little more inclusive and charming than some throwaway ideas we've seen in other games.

As for playtime, Yoshi's Woolly World is pretty generous for a platformer. In an era when many of us want more bang for our buck, Yoshi's Woolly World delivers - rushing through the campaign should take most at least 8-10 hours, but will likely take much longer with co-op sessions and the aforementioned collect-a-thon. Though the basic level count - excluding extra stages - is 48, these can often be lengthy levels full of puzzles and exploration. There are some quicker levels, but some will genuinely stump you at points, and the use of yarn in puzzles that are based on colours or quirky mechanics keep things fresh. As per the Yoshi tradition - especially the classic SNES entry - you go at a slower, more deliberate pace from start to finish than others such as Mario; mad dashes are the exception, not the rule.
Yoshi's Woolly World Review - Screenshot 6 of 6

Of course, with Kirby's Epic Yarn being quite a looker on Wii, thoughts also turn to the visuals of Yoshi's Woolly World. That's a simple judgement from our perspective - it looks gorgeous. There's lovely detail in seeing threads of yarn, environments deliver welcome variety, and there are clever animations and touches that play into Yoshi's ability to transform; Good-Feel shows terrific attention-to-detail and care. The framerate clicks along at a solid 60fps for the most part, too, though we saw a smidge of slowdown in a few moments. Sound is also excellent with some catchy music, though we did notice some tracks appearing a little more regularly than expected.

Yoshi's Woolly World has very few weak spots, though they should be acknowledged. Boss fights, while a visual spectacle, are slightly hit and miss in design and difficulty, with a couple of strong examples let down by repeats or the occasional sense of niggling disappointment. The slow, methodical nature of some stages also makes some of the checkpoints a little too far apart, with moments where you'll be repeating slightly fiddly puzzles before facing a dreaded tricky section. These are rare and relatively minor complaints that do little to detract from the brilliance of the overall product, but deprive it of pure platforming perfection.

### Response:
oshi's Woolly World doesn't break the mold — instead, it polishes it until it shines." Edge Magazine gave the game eight out of ten and stated "There’s nothing ground

</details>

(It fully used up all my gpu vram for the task. It went to the 24gb bar. You can see that the task succeeded, but I have used up all 2048 tokens context length, so it has been interrupted.)

<details><summary> Screenshots: </summary>

Settings: 30b lora, default preset
|:---:|
| ![image](https://user-images.githubusercontent.com/128182951/226213623-c8dfdf0c-eb48-475f-b7e4-47d05f2c274c.png)|
|**Idle vram usage with model loaded**|
|![image](https://user-images.githubusercontent.com/128182951/226213751-9793a7c7-2bc7-44e3-85d6-3e2a7983d796.png)  |

</details>


Another Example with the (### Input) for the wall of text
<details>

Below is an instruction that describes a task. Write a response that appropriately completes the request.

### Instruction:
Provide a 2 sentence summary of the following text:

### Input:
Nintendo has long been the leading light in the platforming genre, a part of that legacy being the focus of Super Mario Anniversary celebrations this year. It's a genre with plenty of scope for variety, however, which is something Nintendo pushed forward with the superb Super Mario World 2: Yoshi's Island on Super NES, placing Mario's sidekick(s) at the core of the adventure with a different approach. Flutter jumping, eating and egg throwing brought a more sedate, puzzle-based approach to running and jumping.

Of course, Kirby was another early innovator in platforming, and was the first of Nintendo's heroes to be turned into yarn by Good-Feel, purveyors of cuteness and clever design. Kirby's Epic Yarn on Wii was a stand-out, and the same core treatment has been applied to Yoshi's Woolly World, with an improved HD engine and a fresh approach. Unlike Kirby's adventure in which the pink one was bolted into what was originally to be a new IP, Yoshi feels like he belongs in this bouncy, soft world; it's a natural fit.

As for storyline, those that would rather not look after Baby Mario should breathe easy, as that mechanic is nowhere to be found here. The plot revolves around a fabric-based world with Kamek appearing and effectively stealing the population of woollen Yoshis, bundling them up into Yarn. Your hero naturally goes in pursuit across a fairly standard stretch of worlds and levels, but it's a cute setup to charm older players and make younger players smile.
Yoshi's Woolly World Review - Screenshot 2 of 6

It's in stage design, utilising all of these factors, that Good-Feel has achieved some wonderful results. We see riffs on toys and yarn in various forms, and the logic (such as it is) quickly embeds itself into gameplay - hurling balls of wool can fill in outlined platforms, while some enemies can only be defeated by wrapping them in yarn with a projectile and following up with a stomp. You see Yoshi's nose squish into fabric when pushing a hidden wall, or you unravel a blocked path with his tongue, while there are stages that play with invisibility and much more besides. A couple of fast-paced stages even brought franchises such as Donkey Kong Country to mind, some are labyrinths and others are focused on verticality. There's some delightfully playful creativity on show.

This also applies to short challenge areas in which Yoshi transforms into various creatures and items, ranging from an agile dolphin, a steady umbrella, a fast motorbike and more besides. These come in short bursts, and some can be missed entirely by inattentive players; nevertheless they provide some truly memorable moments.

Two player local co-op is included, as is the norm with Nintendo platformers, and this feels like one of the stronger uses of the option. It does have an element of playful chaos as you bounce off each other, which is given a delightful malleability due to the woollen effect, but it's also hugely practical for taking on tougher stages or reaching tricky collectibles. This is due to the fact that you can swallow and turn each other into eggs / wool balls, and therefore use the other player as a projectile weapon, carry them through tricky sections or simply hurl them onto an out-of-reach platform. While we believe all collectibles are there to be claimed in single player, co-op is a simple, fun way to reach areas through team work.
Yoshi's Woolly World Review - Screenshot 4 of 6

That could be useful if a skilful player wants to help out a rookie, then, and the Mellow mode is another option. Switchable from Classic mode at any point - even mid-level - this gives Yoshi the ability to fly at will, replacing the frantic flutter jump with a gentle flight. It also slows down the aiming reticule, too, so it's a useful tool.

With progress in the game you also unlock a wide range of Power Badges, with abilities that vary from drawing items to you magnetically, to calling in cute assist character Poochy, making you invincible to fire and more. Though you accumulate plenty of gems in the game, these badges are nevertheless relatively pricey within the in-game economy, and despite the odd freebie the sense is that they're a backup to help you through tough stages or, perhaps, when trying to hunt down elusive flowers.

Some of the Power Badges absolutely become necessary when hunting collectibles. As we mentioned above, completionists will be kept very busy, and as per the Nintendo tradition collecting also unlocks special levels - these come from grabbing all the flowers in each of the six respective worlds. These extra stages are particularly tough, controller throwing tests, pushing capabilities and stripping away reassuring checkpoints. Unlocking and beating these could add a lot of playtime, while grabbing all five balls of Yarn in each level unlocks a neat Yoshi design.
Yoshi's Woolly World Review - Screenshot 5 of 6

More Yoshi designs, some of the best in fact, come through amiibo. Practically all amiibo are supported, even those that seem inconceivable at first - want a Shulk-based Yoshi? No problem. Bowser, or Toad? All included, even the Splatoon inkling. Collecting and playing with different Yoshi designs is certainly enhanced by amiibo. As for the dedicated Yarn Yoshi, we didn't have one for review but it's been shown how this essentially drops in as a clone Yoshi in single player, which makes some aspects of co-op - such as throwing another player to inaccessible areas - possible while playing alone. These are neat extra features utilising the figures, and a little more inclusive and charming than some throwaway ideas we've seen in other games.

As for playtime, Yoshi's Woolly World is pretty generous for a platformer. In an era when many of us want more bang for our buck, Yoshi's Woolly World delivers - rushing through the campaign should take most at least 8-10 hours, but will likely take much longer with co-op sessions and the aforementioned collect-a-thon. Though the basic level count - excluding extra stages - is 48, these can often be lengthy levels full of puzzles and exploration. There are some quicker levels, but some will genuinely stump you at points, and the use of yarn in puzzles that are based on colours or quirky mechanics keep things fresh. As per the Yoshi tradition - especially the classic SNES entry - you go at a slower, more deliberate pace from start to finish than others such as Mario; mad dashes are the exception, not the rule.
Yoshi's Woolly World Review - Screenshot 6 of 6

Of course, with Kirby's Epic Yarn being quite a looker on Wii, thoughts also turn to the visuals of Yoshi's Woolly World. That's a simple judgement from our perspective - it looks gorgeous. There's lovely detail in seeing threads of yarn, environments deliver welcome variety, and there are clever animations and touches that play into Yoshi's ability to transform; Good-Feel shows terrific attention-to-detail and care. The framerate clicks along at a solid 60fps for the most part, too, though we saw a smidge of slowdown in a few moments. Sound is also excellent with some catchy music, though we did notice some tracks appearing a little more regularly than expected.

Yoshi's Woolly World has very few weak spots, though they should be acknowledged. Boss fights, while a visual spectacle, are slightly hit and miss in design and difficulty, with a couple of strong examples let down by repeats or the occasional sense of niggling disappointment. The slow, methodical nature of some stages also makes some of the checkpoints a little too far apart, with moments where you'll be repeating slightly fiddly puzzles before facing a dreaded tricky section. These are rare and relatively minor complaints that do little to detract from the brilliance of the overall product, but deprive it of pure platforming perfection.

### Response:
fore moving forward. And like those games, it doesn't hold your hand during difficult segments, instead trusting you to figure things out on your own. If you don't succeed, well,

</details>

**Comment by yesbroc** [comment]
if u havent already just use superbooga, given this thread is pretty old imma you already know how to use superbooga
